.circleci/config.yml

```diff
@@ -303,6 +303,13 @@ jobs:
           test_targets: "sanity"
   build-linux:
     executor: bionic
+    # xlarge has 4x the cores of the default medium, costs 4x as much, and runs
+    # in about 1/2 the time, so it is not cost-effective (overall it is 2x the
+    # cost for the same work), but given this blocks almost all the other jobs
+    # we want it to finish asap
+    resource_class: xlarge
+    environment:
+      EMCC_CORES: 16
```

Comment:
Why not just remove EMCC_CORES from here and above?  (Or do you want to do that as a separate change?)

Reply:
Yes, I think it's best to do any other changes separately.